src/sandstorm/identity.capnp

```diff
@@ -39,6 +39,13 @@ interface Identity {
 
   # TODO(someday): Public key info? Ability to seal messages / check signatures?
 
+  struct PowerboxTag {
+    # Tag to be used in a `PowerboxDescriptor` to describe an `Identity`.
+
+    identityId @0 :Data;
```

Comment:
There has been some talk about us eventually moving to a model where `identityId` is not global and stable.  I think adding something like `Identity.getId()` or `Profile.identityId` could make such a change trickier.